descartes_light/include/descartes_light/impl/descartes_light.hpp

```diff
@@ -110,9 +114,12 @@ void Solver<FloatType>::build(const std::vector<typename WaypointSampler<FloatTy
     }
 #endif
   }
+  double duration = std::chrono::duration<double, std::milli>(Clock::now() - start_time).count() / 1000.0;
```

Comment:
If you wanted the duration in seconds you can omit the second template argument **std::milli** and it'll default to seconds